content/en/docs/concepts/scheduling/configure-scheduler.md

```diff
@@ -0,0 +1,217 @@
+---
+title: How to Configure Your Kubernetes Scheduler
+reviewers:
+- wgliang
+content_template: templates/concept
+weight: 30
+toc_hide: false
+---
+
+{{% capture overview %}}
+
+This is third article of how to use and extend the Kubernetes scheduler series, the whole series of artices are,
+
+* [How does the Kubernetes scheduler work?](TODO)
+* [How to customize and extend the kubernetes scheduler?](/docs/concepts/scheduling/customization-and-extension/)
+* How to configure your kubernetes scheduler?
+* [Choose scheduling algorithms and strategies for your cluster](TODO)
+* [Learning kube-scheduler's design and implementation from source code.](TODO)
+
+This page explains how to configure Kubernetes scheduler.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Scheduler
+
+The Kubernetes scheduler is one of main components of Kubernetes and responsible for selecting the best node for the pod running against based on a series of predefined scheduler policies. The Kubenetes provides the default scheduler and policies to place the pod one by one to approprorite node in a Kubernetes cluster.
+
+## Configure the Policies
+
+The Kubernetes scheduler plugin frameworks provides the user to 1) Customize the default scheduler policies and 
+2) extend the scheduler or write the new scheduler(s) to either run alongside the default scheduler or replace the default scheduler, please refer to the article [Customization and Extension ](/docs/concepts/scheduling/customization-and-extension/).
```

Comment:
The section needs to be formatted. Reworded the text (minor):

The Kubernetes scheduler plugin frameworks provide the user with the ability to:
1. Customize the default scheduler policies.
2. Extend the scheduler.
3. Write a new scheduler(s) to either run alongside the default scheduler or replace the default scheduler.

For more information, see the article on Customization and Extension.